carve-lockpicks.lic

```diff
@@ -0,0 +1,114 @@
+=begin
+	Need these settings in your yaml:
+		#Stow locations for Grandmaster's, Master's, and anything less.
+    grand_container: backpack
+    master_container: haversack
+    trash_container: cloak
+    #Whether to initial Grandmaster's with an extra carve.
+    initial: true
```

Comment:
A lot of these setting names are kind of generic. A few ideas:

* Rename them to indicate they're for carving lockpicks
* Nest them under a parent setting which indicates they're for carving lockpicks
* Something else?

Reply:
nested under lockpick_carve_settings